static/compatibilities/manifest.yaml

```diff
@@ -14,4 +14,6 @@ names:
 - opentelemetry
 - kyverno
 - vitess
-- strimzi
\ No newline at end of file
+- strimzi
+- keda
+- zalando-postgres
```

Comment:
this name is not going to actually work, I'm not positive what it should be but my guess is `postgres-operator`

Reply:
> this name is not going to actually work, I'm not positive what it should be but my guess is `postgres-operator`

I've changed it to postgres-operator.